autocrafter.lua

```diff
@@ -252,7 +248,7 @@ minetest.register_node("pipeworks:autocrafter", {
 	description = S("Autocrafter"),
 	drawtype = "normal",
 	tiles = {"pipeworks_autocrafter.png"},
-	groups = {snappy = 3, tubedevice = 1, tubedevice_receiver = 1},
+	groups = {snappy = 3, tubedevice = 1, tubedevice_receiver = 1, dig_generic = 1},
```

Comment:
What is `dig_generic` group and where it is used / why it is added?

Reply:
support for other games